app/controllers/api/v1/coupons_controller.rb

```diff
@@ -94,6 +94,9 @@ def input_params
           # NOTE: Legacy field
           :expiration_date,
           :reusable,
+          applies_to: [
+            plan_codes: [],
+          ]
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)